src/sass/Fabric.Icons.Output.scss

```diff
@@ -34,6 +34,10 @@
   @include ms-Icon--l;
 }
 
+// @todo: 'DynamicsCRMLogo' class is deprecated, will be removed in a future release
+// @todo: 'CarotRightSolid8' class is deprecated, will be removed in a future release
```

Comment:
DynamicsCRMLogo is still down there. CarotRightSolid8 is a bit confusing though, as the original code simply had CarotRightSolid8 twice. It wasn't deprecated for something else.